tools/bin/lib/.gitignore

```diff
@@ -2,7 +2,3 @@ netperf
 netserver
 stream
 Crypto
```

Comment:
The tools/bin/Makefile processes the "stream" target and the executable "stream" is copied to tools/bin/lib/stream.  Thus the "stream" entry in the .gitignore file is used.

I will remove the other entries and leave the "stream" reference.